eucaconsole.init

```diff
@@ -46,10 +46,21 @@ PROCESS=eucaconsole
 LOGFILE=/var/log/eucaconsole_startup.log
 PIDFILE=/var/run/eucaconsole/eucaconsole.pid
 NGINXPIDFILE=$(grep nginx.pid /etc/eucaconsole/nginx.conf | awk {'print $2'} | sed s'/\;//')
+MEMCACHEDPIDFILE=/var/run/memcached/eucaconsole-memcached.pid
 CONSOLEUSER=eucaconsole
 
 LOCKFILE=/var/lock/subsys/eucaconsole
 
+# memcached defaults
+USER=memcached
+MAXCONN=1024
+CACHESIZE=64
+OPTIONS=""
```

Comment:
Considering this is eucaconsole's init script, not memcached's most of these variables have nowhere near the effects that their names suggest.